firebase-installations-interop/src/main/java/com/google/firebase/installations/FirebaseInstallationsApi.java

```diff
@@ -51,4 +52,12 @@
    */
   @NonNull
   Task<Void> delete();
+
+  /**
+   * Register a listener to receive fid changes.
```

Comment:
Should there be a way to "unregister" the listener? A common way to do it is to have the `register` method return a "registration" object, that has a `remove()` method.

Reply:
Done. PTAL if the changes look.